### ISSUE TITLE
Eng - Ru switching server config

### ISSUE THREAD

**Issue by TheShuEd** [crystallpunk-14/crystall-punk-14#173]
We need to put the language of the server in the configs: so that it is possible to have servers with different languages, differing only by the config

**Comment by c4llv07e** [comment]
There are some problems with this - client loads culture info before connecting to the server (for the title screen text, tips, etc.) so it would require to reload `ContentLocalizationManager` initialization after connection. But there is no way to reload it. Game code can't even check if a specific culture info was loaded or not, and if you try to load it again, game will crash.
But besides that, it is working. Even with the above problems, after wrong localization prediction it shows correctly localized text.

**Comment by TheShuEd** [comment]
Yes, I've already experienced this problem. However, I have heard that someone managed to implement it. So there is a way, we just haven't found it yet.

**Comment by c4llv07e** [comment]
I think they either changed client's engine or made it with a large number of assumptions, because wizards' client part can't handle blind localization reloading at all.
It is sad that we can't allow players to select any language as they want because it will break prediction.
